lib/structures/ForumChannel.ts

```diff
@@ -25,7 +25,7 @@ import TypedCollection from "../util/TypedCollection";
 import type { ChannelTypes, ThreadAutoArchiveDuration } from "../Constants";
 import { AllPermissions, Permissions, SortOrderModes } from "../Constants";
 
-/** Represents a forum channel. Documentation for these is currently scarce, so they may not work entirely correctly. */
+/** Represents a forum channel. Documentation for these are currently scarce, so they may not work entirely correctly. */
```

Comment:
```suggestion
/** Represents a forum channel. Documentation for these is currently scarce, so they may not work entirely correctly. */
```

`is` is correct here